dotCMS/src/main/java/com/dotcms/content/elasticsearch/util/ESMigrationUtil.java

```diff
@@ -46,15 +48,21 @@ public void migrateStructure(Structure struct) throws DotDataException, DotSecur
 			if (cons.size() == 0) {
 				break;
 			}
-			Client client = new ESClient().getClient();
-			BulkRequestBuilder bulkRequest = client.prepareBulk();
+
+
+			final BulkRequest request = new BulkRequest();
+			request.timeout(TimeValue.timeValueMillis(INDEX_OPERATIONS_TIMEOUT_IN_MS));
+
 			for (Contentlet c : cons) {
 
 				//bulkRequest.add(client.prepareIndex(ESIndexAPI.ES_INDEX_NAME, type, c.getInode()).setSource(
 				//		new ESMappingAPIImpl().toJson(c)));
 
 			}
-			BulkResponse bulkResponse = bulkRequest.execute().actionGet(INDEX_OPERATIONS_TIMEOUT_IN_MS);
+
+			BulkResponse bulkResponse = Sneaky.sneak(() -> RestHighLevelClientProvider.getInstance().getClient()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'bulkResponse' could be declared final](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=3712629)